.gitignore

```diff
@@ -0,0 +1,4 @@
+.sass-cache/
```

Comment:
### `REQUIRED`

`.sass-cache/` is the right rule for ignoring the `.sass-cache` directory :file_folder: :heavy_check_mark:  but **Git will not ignore the file if you add a rule later.** This means you have committed this folder before adding the rule and `.gitignore` can ignore it anymore. :disappointed: But wait... :stop_sign: There's a solution to this... :smile: 

The solution is to `untrack` the file first. You can do this by running this command:

```bash
git rm -r --cached .sass-cache
```
Now `.gitignore` will recognize the folder and when you commit your changes the file won't be included. :ok_hand: 

You can learn more about this [here](https://help.github.com/en/articles/ignoring-files).